book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Let's take a look at our Rental class, you will notice that it takes three parameters. I have noticed that you are not instantiating the class Rental in this Book class. The rental class can be called, you can do something like this to pass the parameters `Rental.new(date, self, person)` `self` in this case represents a book object. You should remember that we are renting a book to a person. This [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) can help solve this issue.